service/service.go

```diff
@@ -27,6 +27,11 @@ import (
 	"github.com/giantswarm/rbac-operator/service/controller/clusternamespace"
 	"github.com/giantswarm/rbac-operator/service/controller/crossplane"
 	"github.com/giantswarm/rbac-operator/service/controller/rbac"
+
+	"k8s.io/apimachinery/pkg/runtime"
+	clientgoscheme "k8s.io/client-go/kubernetes/scheme"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/log/zap"
```

Comment:
Nit, but imports should follow the [import order](https://github.com/giantswarm/fmt/blob/main/go/imports.md) from the style guide